.circleci/config.yml

```diff
@@ -29,7 +29,7 @@ commands:
           name: Unit tests
           command: |
             mkdir -p reports
-            $NYC yarn test --reporter mocha-junit-reporter
+            $NYC yarn test
```

Comment:
Do we lose the junit output entirely? It's nice to use Circles reporting from the junit reporter

Reply:
Unfortunately with mocha, you can technically only have a single reporter. A junit file reporter for Circle failure reporting or being able to see successful CI output using the default reporter?

We could implement a third-party reporter like [mocha-multi-reporters](https://www.npmjs.com/package/mocha-multi-reporters), but I thought that could be another PR.

Reply:
Yeah that could be another PR. Wanna create a clubhouse / GH issue (whichever)?